requirements/linting.txt

```diff
@@ -2,7 +2,7 @@
 # This file is autogenerated by pip-compile with Python 3.10
 # by the following command:
 #
-#    pip-compile --output-file=requirements/linting.txt --resolver=backtracking requirements/linting.in
```

Comment:
I ran the command with the `--resolver=backtracking` flag but the output was still written as it is.